openmc/element.py

```diff
@@ -0,0 +1,208 @@
+from collections import OrderedDict
+import re
+import os
+from xml.etree import ElementTree as ET
+
+import openmc
+import openmc.checkvalue as cv
+from openmc.data import NATURAL_ABUNDANCE, atomic_mass
+
+
+class Element(str):
+    """A natural element that auto-expands to add the isotopes of an element to
+    a material in their natural abundance. Internally, the OpenMC Python API
+    expands the natural element into isotopes only when the materials.xml file
+    is created.
+
+    Parameters
+    ----------
+    name : str
+        Chemical symbol of the element, e.g. Pu
+
+    Attributes
+    ----------
+    name : str
+        Chemical symbol of the element, e.g. Pu
+
+    """
+
+    def __new__(cls, name):
+        cv.check_type('element name', name, str)
+        cv.check_length('element name', name, 1, 2)
+        return super().__new__(cls, name)
+
+    @property
+    def name(self):
+        return self
+
+    def expand(self, percent, percent_type, enrichment=None,
+               cross_sections=None):
+        """Expand natural element into its naturally-occurring isotopes.
+
+        An optional cross_sections argument or the OPENMC_CROSS_SECTIONS
```

Comment:
Recommend putting OPENMC_CROSS_SECTIONS in code-quotes \`OPENMC_CROSS_SECTIONS\` so that it is clear that it is a variable.